tensorflow/lite/CMakeLists.txt

```diff
@@ -70,6 +70,9 @@ option(TFLITE_ENABLE_GPU "Enable GPU" OFF)
 # This is currently not supported.
 option(TFLITE_ENABLE_FLEX "Enable SELECT_TF_OPS" OFF) # TODO: Add support
 option(TFLITE_ENABLE_XNNPACK "Enable XNNPACK backend" ON)
+
+option(TFLITE_KERNEL_TEST "Enable tflite kernel unit test" ON)
```

Comment:
Shall we use "OFF" for default?

Reply:
@terryheo 
Does tensorflow have cmake-base ci test?
If yes, is it better to run the kernel tests for cmake build?

Reply:
Yes we have. But it's different issue. We can always control this by providing the CMake argument.
I think most users don't want to build it by default.

Reply:
Oh.. Let's control it by the cmake argument.